### PR TITLE
Bring up to date

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Update the media consent form as required in https://github.com/srobo/tasks/issues/187.

Each commit is atomic, and contains the notable changes.

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
This PR has, in its current state, been reviewed and approved by the trustees.

**Comment by PeterJCLaw** [comment]
> This PR has, in its current state, been reviewed and approved by the trustees.

Cool. Just to be extra clear on this: that includes the consent age being 16?

File: Makefile [hunk 1]
```diff
@@ -3,14 +3,10 @@ all: form.pdf
 %.pdf: %.svg
 	inkscape -A $@ $<
 
-marketing/logos:
```
Comment (PeterJCLaw) [inline Makefile:6]: Why are we _removing_ this target?
I appreciate that the submodule has changed, however I still think it's useful to have a target of this form.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Media Consent
+
+[![CircleCI](https://circleci.com/gh/srobo/media-consent.svg?style=svg)](https://circleci.com/gh/srobo/media-consent)
+
+PDFs can be downloaded from the 'Artifacts' tab in CircleCI.
```
Comment (PeterJCLaw) [inline README.md:5]: This tab only shows up if you're logged in. Is this expected? Should we note this?
For this repo I think it's fine if the artefacts are public, though I don't think it's a big deal (a comment here would suffice for now IMO).

**Review by PeterJCLaw — APPROVED** [review]

**Comment by PeterJCLaw** [comment]
Once merged, please could you also update the [media-consent-access](https://www.studentrobotics.org/cgit/media-consent-access.git) repo as well.

**Comment by RealOrangeOne** [comment]
> > This PR has, in its current state, been reviewed and approved by the trustees.
> 
> Cool. Just to be extra clear on this: that includes the consent age being 16?

Correct

**Pull request merged**